ui/src/components/home/EventsSection.js

```diff
@@ -70,83 +134,117 @@ const EventsSection = () => {
-      <Container maxW="container.lg" justifyContent="center" display="flex">
-        <VStack>
-          <FeaturedEvent />
-        </VStack>
-      </Container>
+    <Flex
+      w="full"
+      h="auto"
+      direction="column"
+      background="#172848"
+      alignItems="center"
+      paddingTop="2em"
+      paddingBottom="2em"
+      boxSizing=""
+    >
       <Container
         maxW="container.lg"
         justifyContent="center"
         display="flex"
         marginTop="2em"
+        ref={marginRef}
       >
-        <VStack w="full" alignItems={["flex-start", null]}>
+        <VStack w="full" alignItems={['flex-start', null]}>
           <HStack
-            w={[null, "full"]}
+            w={[null, 'full']}
             whiteSpace="nowrap"
             height="10vh"
             marginBottom="1em"
+            justifyContent="space-between"
           >
-            <Heading fontSize={["2em", "5xl"]} color="black">
-              Upcoming Events
-            </Heading>
-            <chakra.hr
-              width="full"
-              color="black"
-              border="none"
-              height="2px"
-              bgColor="black"
-              display={["none", "block"]}
-            />
-            <Text color="black" display={["none", "block"]} fontWeight="900">
-              <Link href="/events">{allEventsText}</Link>
-            </Text>
+            <VStack alignItems="flex-start">
+              <Heading fontSize={['2em', '4xl']} fontWeight={600} color="white">
+                Events
+              </Heading>
+              <Text color="white">
+                Check out what's happening in HMCC of Hong Kong
+              </Text>
+            </VStack>
+            <Link
+              href="/events"
+              textDecoration="none"
+              _hover={{ textDecoration: 'none' }}
+            >
+              <Button
+                color="#A5CBFF"
+                background="transparent"
+                border="2px solid #A5CBFF"
+                borderRadius="7px"
+                display={{ base: 'none', md: 'block' }}
+                fontWeight="700"
+              >
+                {allEventsText}
+              </Button>
+            </Link>
           </HStack>
         </VStack>
       </Container>
       <Box
-        w="full"
+        w="100%"
+        boxSizing="border-box"
         display="flex"
+        paddingLeft={computedMargin}
         justifyContent="flex-start"
+        alignItems="flex-start"
         height="auto"
-        overflowX={["auto", "auto", "auto", "auto", "hidden"]}
-        overflowY="hidden"
+        overflowX={['auto', 'auto', 'auto', 'auto', 'hidden']}
         whiteSpace="nowrap"
-        marginBottom={["none", "1.5em"]}
+        marginTop="2.5em"
+        marginBottom={['none', '3em']}
         _hover={{
-          overflowX: "auto",
+          overflowX: 'auto',
         }}
       >
         <Slider {...sliderSettings} style={sliderStyle}>
```

Comment:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/24607458/184651628-aa98f09e-d3d9-4cfc-be98-905d75712ae3.png">
currenty the event cards get cut off as we scroll through the slider. in the current website, as you scroll the cards take up the whole width of the container. 
<img width="957" alt="image" src="https://user-images.githubusercontent.com/24607458/184651971-37a1b848-d07d-4881-9cb3-64f24b182eed.png">